doc/nrf/releases/release-notes-changelog.rst

```diff
@@ -142,7 +142,10 @@ nRF9160: Asset Tracker v2
 nRF9160: Serial LTE modem
 -------------------------
 
-|no_changes_yet_note|
+* Removed:
+
+  * DFU AT commands #XDFUGET, #XDFUSIZE and #XDFURUN are removed.
+    Commands are not usable without a custom application in the target (nRF52 series) device.
```

Comment:
Is this explaining why the commands were removed, or clarifying that they can now only be used by having a custom application?

Reply:
This is trying to explain why the commands **were** removed.